src/Http/Http.Extensions/gen/RequestDelegateGeneratorSources.cs

```diff
@@ -19,7 +19,97 @@ internal static class RequestDelegateGeneratorSources
 
     public static string GeneratedCodeAttribute => $@"[System.CodeDom.Compiler.GeneratedCodeAttribute(""{typeof(RequestDelegateGeneratorSources).Assembly.FullName}"", ""{typeof(RequestDelegateGeneratorSources).Assembly.GetName().Version}"")]";
 
-    public static string GetGeneratedRouteBuilderExtensionsSource(string genericThunks, string thunks, string endpoints) => $$"""
+    public static string TryResolveBodyAsyncMethod => """
+        private static async ValueTask<(bool, T?)> TryResolveBodyAsync<T>(HttpContext httpContext, bool allowEmpty)
```

Comment:
Should these strings be cut over to CodeWriter implementations. Given where these get emitted, we can pretty easily predict the indentation requirements but really just asking for the sake of consistency since they are multi-line blocks.